plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -146,14 +146,15 @@ function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $b
  *
  * @since 1.4.0
  *
- * @param int                    $id            The image attachment post ID.
- * @param string|array{int, int} $size          Image size name or array of width and height.
- * @param string                 $align         The image alignment.
- * @param int                    $resize_width  Resize image width.
- * @param string                 $max_alignment The maximum usable layout alignment.
+ * @param int                    $id                       The image attachment post ID.
+ * @param string|array{int, int} $size                     Image size name or array of width and height.
+ * @param string                 $align                    The image alignment.
+ * @param int                    $resize_width             Resize image width.
+ * @param string                 $max_alignment            The maximum usable layout alignment.
+ * @param float                  $container_relative_width Container relative width.
```

Comment:
Related to the above, is this expecting a `float` or a float in a `string`?

Reply:
Yes it's float value that update in https://github.com/WordPress/performance/pull/1818/commits/90dc20fa67fad5df900368bd31ebec6f0ac09b2b